docs/pages/reference/cli.mdx

```diff
@@ -484,6 +590,45 @@ This command does not accept any arguments.
 These flags are available for all commands `--login, --proxy, --user, --ttl, --identity, --cert-format, --insecure, --auth, --skip-version-check, --debug, --jumphost, --format`.
 Run `tsh help <subcommand>` or see the [Global Flags Section](#tsh-global-flags)
 
+### tsh proxy aws
+
+Start a local proxy for AWS access. The user must already be logged into at
```

Comment:
```suggestion
Start a local proxy for AWS access. The user must already be logged in to at
```

[reference](https://www.grammarly.com/blog/into-vs-in-to/#:~:text=I%20dropped%20into,to%20say%20hello.)